fizzbuzz.js

```diff
@@ -0,0 +1,10 @@
+
+for (var i = 1; i <= 100; i++) {
```

Comment:
昨今では `var` はほとんど使われませんので `let` を使うようにしてください。
```suggestion
for (let i = 1; i <= 100; i++) {
```